plugins/audit_logs/scripts/report.py

```diff
@@ -0,0 +1,97 @@
+"""
+This script is for creating download reports for a given folder.
+
+    girder shell --plugins=audit_logs report.py -- --folder=57557fac8d777f68be8f3f49 --start-date=2018-09-10T13:55:34.847Z --end-date=2018-09-13T13:55:34.847Z report.csv
+"""
+import click
+import datetime
+import dateutil.parser
+
+from bson.objectid import ObjectId
+
+from girder.models.item import Item
+from girder.models.folder import Folder
+from girder.constants import AccessType
+from girder.plugins.audit_logs import Record
+
+
+def index_folder(folderId):
+    """
+    Return a list of ObjectId representing every item in a folder recursively.
+    """
+    if Folder().load(folderId, level=AccessType.READ) is None:
+        raise ValueError('folderId={} was not a valid folder'.format(folderId))
+    items = Item().find({'folderId': ObjectId(folderId)})
+    subfolders = Folder().find({'parentId': ObjectId(folderId)})
+
+    files = {}
+    for item in items:
+        itemdoc = Item().load(item['_id'], level=AccessType.READ)
+        for file in Item().fileList(itemdoc, data=False):
```

Comment:
`Item().childFiles` is probably less awkard since it just returns a cursor of file documents. And you can pass `fields={'_id': True}` to that since that's the only thing we'll need for now.